packages/extension/src/pages/chat/index.tsx

```diff
@@ -163,9 +178,9 @@ const ChatView = () => {
       rightRenderer={<SwitchUser />}
     >
       <div className={style.chatContainer}>
-        {/* {!user.accessToken && (
+        {open && openDialog && messagingPublicKey.length == 0 && (
```

Comment:
if we are dependent on this messagingPublicKey.length then we should not need open and opendialog

Reply:
I will fix this.